### ISSUE TITLE
Package request: ord

### ISSUE THREAD

**Issue by taxmeifyoucan** [debian-cryptoanarchy/cryptoanarchy-deb-repo-builder#217]
**Did you check if Debian 10 already contains this software?**
Yes, it does not.

**What the application you want packaged does?**
https://github.com/casey/ord
_Ordinal theory imbues satoshis with numismatic value, allowing them to collected and traded as curios._
Connects to bitcoind and serves as an index and wallet for inscriptions. 

**How does the application relate to the ideas of privacy and freedom?**
It extends all features enabled by fungible Bitcoin to non fungible use cases as well. 

**Alternative software**
Alternative would be another inscription software I am not aware of or a completely different technology enabling similar features, e.g. RSK, but that comes with many tradeoffs. 

**Integrations**
Which packages need/should be integrated with this software? (write `x` into square brackets or add your own)
- [ ] `selfhost`, `selfhost-dahsboard` (all web apps)
- [ x] `bitcoin-*`
- [ ] `lnd-*`

**Comment by Kixunil** [comment]
> ```toml
> rust-version = "1.67"
> ```

Nope, at least not yet. I'm considering setting up alternative Rust compiler with full verification but that's not very important to me and will take time. FTR Bookworm only has 1.63.
